flowzone.yml

```
@@ -1398,7 +1422,38 @@ jobs:
           echo "branch_tag=${branch_tag}" >> $GITHUB_OUTPUT
           echo "sha_tag=${sha_tag}" >> $GITHUB_OUTPUT
           echo "version_tag=${version_tag}" >> $GITHUB_OUTPUT
+  ###################################################
+  ## website
+  ###################################################
 
+  website_build:
+    name: Build website using Docusaurus
+    runs-on: ${{fromJSON(inputs.runs_on)}}
+    needs: [project_types,versioned_source]
+    if: |
+      needs.project_types.outputs.cloudflare_website == 'true' &&
+      inputs.cloudflare_website != ''
+    steps:
+      - uses: actions/checkout@v3
```

This should be using
```suggestion
      - *downloadSourceArtifact
      - *extractSourceArtifact
```
so that the versioned source is what is used